fhir-persistence/src/main/java/com/ibm/fhir/persistence/index/TokenParameter.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * (C) Copyright IBM Corp. 2022
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+ 
+package com.ibm.fhir.persistence.index;
+
+
+/**
+ * A token search parameter value
+ */
+public class TokenParameter extends SearchParameterValue {
+    private String valueSystem;
+    private String valueCode;
+
+    // for storing versioned references
+    private Integer refVersionId;
```

Comment:
do our token-values tables support versioning?

Reply:
Good question. Perhaps now that references are moved to xx_ref_values, we can get rid of the version columns in the schema too.